### ISSUE TITLE
Cannot link VS Code in OSX to VSTS

### ISSUE THREAD

**Issue by erwindemoel** [Microsoft/vsts-vscode#421]
I'm trying to connect VS Code in OSX to our code repository in Visual Studio Team Services.

Machine information:
OS: Mac OS X High Sierra
java version "10.0.2" 2018-07-17
Java(TM) SE Runtime Environment 18.3 (build 10.0.2+13)
Java HotSpot(TM) 64-Bit Server VM 18.3 (build 10.0.2+13, mixed mode)
VS Code Version 1.25.1 (1.25.1)
TEE-CLC-14.134.0

I've followed the tutorial and also the suggestions in this post by @jeffyoung :
https://github.com/Microsoft/vsts-vscode/issues/174

What I have done is create a workspace using:
`tf workspaces -collection:https://ourURL.visualstudio.com`

Then created a local workspace using:
`tf workspace -new -collection:https://ourURL.visualstudio.com -location:local MI-Workspace`

Then created a local folder mapping using:
`tf workfold -map -collection:https://ourURL.visualstudio.com -workspace:MI-Workspace $/MyFirstProject .`

When I run the command `tf get -all` from the workspace directory MI-Workspace, I get the message:
`An argument error occurred: The workspace could not be determined from any argument paths or the current working directory.`

When I run the command `tf workfold -workspace:MI-Workspace`, I get the message:
```
================================================================================================================================
Workspace:  MI-Workspace
Collection: https://ourURL.visualstudio.com/
```

So there is a workspace and mapping created. When I run the command `tf workspaces`, I get the message:
```
Collection: https://ourURL.visualstudio.com/
Workspace    Owner         Computer    Comment
------------ ------------- ----------- -----------------------------------------------------------------------------------------
MI-Workspace Erwin de Moel Erwins-iMac 
```

When I open the folder in VS Code, I get the message:
```
(team) No Team Services or Team Foundation Server repository configuration was found. Ensure you've opened a folder that contains a repository.
```

When running `ls -a` from the MI-Workspace folder, there also isn't a .tf file present. It's completely empty and returns:
`.    ..`

What am I doing wrong?

**Comment by mikejamesowen** [comment]
I am having the same issue.  Did you ever discover what the problem was?

Thanks

**Comment by erwindemoel** [comment]
Yes, I simply clicked on Clone in VS Code from Azure DevOps (top right menu)

**Comment by mikejamesowen** [comment]
Thanks for the quick answer!  Unfortunately whist I think I have created a Workspace using the:
tf workspace -new ....
and
tf workfold -map ....
commands as there were no errors, when I search for the workspace file (?) to open it from VS Code I have no idea where to find it.  I even searched 'code-workspace' in Finder, and whist it found a GIT workspace that I had created it didn't seem to find the new Azure DevOps one that I created.  I assume that from within VS Code I should be using 'File/Open Workspace' and  trying to locate this file?

If so, even though like you I didn't get any errors when I run the:
tf workfold -workspace:b2b-workspace

command.

Hmmmm

**Comment by mikejamesowen** [comment]
Ah, I think I may have found one of the issues - When I restarted VS Code it asked me if I wanted to 'Set Up TFVC Support....', when I clicked on it it told me that '(team) Unable to find tfvc executable'.

I checked and I had put the 'full path' as it said, i.e. '/usr/local/Cellar/tee-clc/14.134.0/bin', but not the actual filename '/usr/local/Cellar/tee-clc/14.134.0/bin/tf'.  Once I had corrected this, this error went away.  Still not seeing the 'Azure DevOps' top  right menu though.
